_posts/2015-06-26-ux-part-1.md

```diff
@@ -0,0 +1,69 @@
+
+The primary users of OME are bench Life Scientists, but our user base has rapidly expanded to include digital pathology, teaching, publishing and a number of other domains. This expansion is continual and open-ended. But OME does not just have the people who use the clients, scientists and so on, as users, we also have facility managers and system administrators in institutions using OMERO. 
+
+Then in addition to this, as an open source project, OME also effectively has a lot of other developers as users of the system. And to add another layer, being open source, this is very much a two-way street, so the developer-users are also part of the team as well as being users. A slightly tangled web but very fruitful if handled well!
+
+The focus of this series of posts is on the scientist user and the GUI clients, OMERO.insight and OMERO.web, where the primary UX team involvement is. 
+
+It may sound obvious, but the client user interface is the only way the majority of users have to interact with any computer system. No matter how wonderful the system is behind the scenes, the typical user only gets to use what they can see and interact with via the user interface. So in effect, the UI ultimately determines the functionality of the software for most users.
+
+In the case of OME, in its early days as an expert system, it enabled users to access functionality directly through its Application Program Interface (API) via scripts or the command line, so there is a huge legacy of functionality in the system that has never been exposed via the GUI clients. Therefore a key part of the UX team’s task is to identify and help incorporate new workflows into the UIs that expose currently unavailable functionality, without significant disruption to any existing workflows.
+
+The OME team is lucky to benefit from the broad range of backgrounds of its members, from experience as biological scientists, through facility management, medicine, maths  and physics. Direct experience of the domains in which the software is used helps enormously when dealing with users and understanding their requirements, tasks and workflows.
+
+##User-Centred Development
+
+There are a number of classical models of software development in the HCI world. OME probably comes closest to Hartson and Hix’s  Star Model, and although we have dabbled with Agile a bit, it is probably not the best model when there is no monolithic “client” for the software. However, this is not the place to delve into the details of user-centred design, but rather just pick up a couple of points, then show the process used in OME.
```

Comment:
do you need "client" as in customer here? It's a bit confusing given elsewhere client is used to mean insight or web. I'd be inclined to use customer instead.